src/cplscheme/tests/ParallelImplicitCouplingSchemeTest.cpp

```diff
@@ -120,7 +120,7 @@ BOOST_AUTO_TEST_CASE(testInitializeData)
 
   // Create the coupling scheme object
   ParallelCouplingScheme cplScheme(
-      maxTime, maxTimesteps, timestepLength, 16, nameParticipant0, nameParticipant1,
+      maxTime, maxTimeSteps, timeStepSize, 16, nameParticipant0, nameParticipant1,
```

Comment:
```suggestion
      maxTime, maxTimeWindows, timeWindowSize, 16, nameParticipant0, nameParticipant1,
```
These correspond 1:1 to the values defined in the configuration, no?

Reply:
Exactly. For example [here](https://github.com/precice/precice/blob/691fccdc406f6c58d4c222da5622bc26144a1b2f/src/cplscheme/config/CouplingSchemeConfiguration.cpp#L828-L831). As far as I see it we already fixed this in the config previously.

Reply:
Fixed it in 09bdd761e72774a0d440cc0fef0b2671fcaca835.